### DISCUSSION TITLE
Application Error When Switching Language on Next.js Documentation

### DISCUSSION THREAD

**Discussion by Motonosuke** [vercel/community#2445] [Bug Report]
I've encountered an issue on the Next.js docs site (https://nextjs.org/docs).

Steps to Reproduce:

1. Right-click on the page and change the language(en to ja).
2. Confirm that the language has been successfully changed.
3. Click on the "Defining Routes" option under the Routing accordion in the left sidebar.
4. An application error message is displayed.

This problem seems to occur not only on the Defining Routes page, but also on the Pages and Layouts page, etc., after switching languages.

The console logs the following errors:

- DOMException: Failed to execute 'insertBefore' on 'Node': The node before which the new node is to be inserted is not a child of this node.
- DOMException: Failed to execute 'removeChild' on 'Node': The node to be removed is not a child of this node.

I hope this information is helpful in resolving the bug. Please let me know if additional details are needed.

https://github.com/vercel/community/assets/67644458/09addd08-784d-4f61-9c72-8d1041390839

**Comment by amyegan** [comment]
Thank you for sharing this, @Motonosuke. I shared it, and someone from the docs team may reach out if they need more info
